### ISSUE TITLE
list displays not in watchers shouldn't be editable

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap--Build-Your-Own-Blocks#918]
If you click on an expression in the scripting area whose value is a list, you of course get a speech balloon containing a list.  That list picture shouldn't be editable, because you can't change the value reported by the expression and it's confusing if you try.  Only variable watchers on stage whose value is a list should be editable.

**Comment by cycomachead** [comment]
FWIW this came up in PD last week with Dan.

**Comment by jmoenig** [comment]
Yeah, I emphatically disagree. All data is mutable. Always and everywhere.

**Comment by cycomachead** [comment]
A reporter's data is not "mutable"! It's not even stored... It simply doesn't do anything meaningful.

What purpose does an action like this serve?
<img width="306" alt="screen shot 2016-07-19 at 2 06 33 pm" src="https://cloud.githubusercontent.com/assets/1505907/16966613/33256dc2-4dba-11e6-8e51-d6218329383e.png">
